.eslintrc.js

```diff
@@ -71,6 +72,12 @@ module.exports = {
       },
     },
 
+    // test files
+    // {
+    //   files: ['tests/**/*-test.{js,ts}'],
+    //   extends: ['plugin:qunit/recommended'],
+    // },
+
```

Comment:
Newer Ember apps and addons come with `eslint-plugin-qunit` installed. Several tests fail linting so I disabled the addon. Let's fix the tests in a separate pull request.